src/main/java/com/mraof/minestuck/data/tag/MinestuckBlockTagsProvider.java

```diff
@@ -150,20 +150,11 @@ protected void addTags(HolderLookup.Provider provider)
-				TREATED_UNCARVED_WOOD_BUTTON.get(), TREATED_UNCARVED_WOOD_FENCE.get(), TREATED_UNCARVED_WOOD_FENCE_GATE.get(), TREATED_UNCARVED_WOOD_PRESSURE_PLATE.get(), TREATED_UNCARVED_WOOD_SLAB.get(),
-				TREATED_UNCARVED_WOOD_STAIRS.get(), TREATED_CHIPBOARD.get(), TREATED_CHIPBOARD_BUTTON.get(), TREATED_CHIPBOARD_FENCE.get(), TREATED_CHIPBOARD_FENCE_GATE.get(), TREATED_CHIPBOARD_PRESSURE_PLATE.get(),
-				TREATED_CHIPBOARD_SLAB.get(), TREATED_CHIPBOARD_STAIRS.get(), TREATED_HEAVY_PLANKS.get(), TREATED_HEAVY_PLANK_SLAB.get(), TREATED_HEAVY_PLANK_STAIRS.get(), TREATED_PLANKS.get(), TREATED_BUTTON.get(),
-				TREATED_FENCE.get(), TREATED_FENCE_GATE.get(), TREATED_PRESSURE_PLATE.get(), TREATED_PLANKS_SLAB.get(), TREATED_PLANKS_STAIRS.get(), TREATED_CARVED_KNOTTED_WOOD.get(), POLISHED_TREATED_UNCARVED_WOOD.get(),
-				POLISHED_TREATED_UNCARVED_SLAB.get(), POLISHED_TREATED_UNCARVED_STAIRS.get(), TREATED_WOODEN_GRASS.get(), LACQUERED_UNCARVED_WOOD.get(), LACQUERED_UNCARVED_WOOD_BUTTON.get(), LACQUERED_UNCARVED_WOOD_FENCE.get(),
-				LACQUERED_UNCARVED_WOOD_FENCE_GATE.get(), LACQUERED_UNCARVED_WOOD_PRESSURE_PLATE.get(), LACQUERED_UNCARVED_WOOD_SLAB.get(), LACQUERED_UNCARVED_WOOD_STAIRS.get(), LACQUERED_CHIPBOARD.get(), LACQUERED_CHIPBOARD_BUTTON.get(),
-				LACQUERED_CHIPBOARD_FENCE.get(), LACQUERED_CHIPBOARD_FENCE_GATE.get(), LACQUERED_CHIPBOARD_PRESSURE_PLATE.get(), LACQUERED_CHIPBOARD_SLAB.get(), LACQUERED_CHIPBOARD_STAIRS.get(), LACQUERED_HEAVY_PLANKS.get(),
-				LACQUERED_HEAVY_PLANK_SLAB.get(), LACQUERED_HEAVY_PLANK_STAIRS.get(), LACQUERED_PLANKS.get(), LACQUERED_BUTTON.get(), LACQUERED_FENCE.get(), LACQUERED_FENCE_GATE.get(), LACQUERED_PRESSURE_PLATE.get(),
-				LACQUERED_SLAB.get(), LACQUERED_STAIRS.get(), LACQUERED_CARVED_KNOTTED_WOOD.get(), POLISHED_LACQUERED_UNCARVED_WOOD.get(), POLISHED_LACQUERED_UNCARVED_SLAB.get(), POLISHED_LACQUERED_UNCARVED_STAIRS.get(), LACQUERED_WOODEN_MUSHROOM.get(), WOODEN_LAMP.get());
-		tag(NEEDS_STONE_TOOL).add(UNCARVED_WOOD.get(), UNCARVED_WOOD_BUTTON.get(), UNCARVED_WOOD_FENCE.get(), UNCARVED_WOOD_FENCE_GATE.get(), UNCARVED_WOOD_PRESSURE_PLATE.get(), UNCARVED_WOOD_SLAB.get(),
+		tag(MINEABLE_WITH_AXE).add(CARVED_PLANKS.get(), CARVED_BUTTON.get(), CARVED_FENCE.get(), CARVED_FENCE_GATE.get(), CARVED_PRESSURE_PLATE.get(), CARVED_SLAB.get(), CARVED_STAIRS.get(), WOODEN_GRASS.get(),
+				TREATED_PLANKS.get(), TREATED_BUTTON.get(), TREATED_FENCE.get(), TREATED_FENCE_GATE.get(), TREATED_PRESSURE_PLATE.get(), TREATED_PLANKS_SLAB.get(), TREATED_PLANKS_STAIRS.get(),
+				TREATED_WOODEN_GRASS.get(), LACQUERED_PLANKS.get(), LACQUERED_BUTTON.get(), LACQUERED_FENCE.get(), LACQUERED_FENCE_GATE.get(), LACQUERED_PRESSURE_PLATE.get(),
+				LACQUERED_SLAB.get(), LACQUERED_STAIRS.get(), LACQUERED_WOODEN_MUSHROOM.get(), WOODEN_LAMP.get());
+		needsStoneAxe(UNCARVED_WOOD.get(), UNCARVED_WOOD_BUTTON.get(), UNCARVED_WOOD_FENCE.get(), UNCARVED_WOOD_FENCE_GATE.get(), UNCARVED_WOOD_PRESSURE_PLATE.get(), UNCARVED_WOOD_SLAB.get(),
```

Comment:
The way it is now has no net effect in data files. I think it helps illustrate some peculiarities/inconsistencies with how the tags are handled for these blocks